### DISCUSSION TITLE
Don't Know password

### DISCUSSION THREAD

**Discussion by Login-out** [slntopp/nocloud#1200] [Q&A]
I have successfully installed through docker-compose, but I can't seem to find the username and password for web login

**Comment by Login-out** [comment]
sorry，Already able to log in, but an error message appears
"logger":"Token","msg":"Failed to store session","error":"dial tcp: lookup redis on 127.0.0.11:53: server misbehaving

**Reply by slntopp** [reply]
Hi, could you please share where do you get this error and what are you trying to do? :)
